packages/@react-spectrum/tag/test/TagGroup.test.js

```diff
@@ -299,6 +299,85 @@ describe('TagGroup', function () {
     expect(onRemoveSpy).toHaveBeenCalledWith('1');
   });
 
+  it('maxRows should limit the number of tags shown', function () {
+    let offsetWidth = jest.spyOn(HTMLElement.prototype, 'getBoundingClientRect').mockImplementationOnce(function () {
+      return {
+        left: 0,
+        top: 0,
+        x: 0,
+        y: 0,
+        width: 100,
+        height: 50
+      };
+    }).mockImplementationOnce(function () {
+      return {
+        left: 0,
+        top: 0,
+        x: 0,
+        y: 0,
+        width: 100,
+        height: 50
+      };
+    }).mockImplementationOnce(function () {
+      return {
+        left: 0,
+        top: 0,
+        x: 0,
+        y: 100,
+        width: 100,
+        height: 50
+      };
+    }).mockImplementationOnce(function () {
+      return {
+        left: 0,
+        top: 0,
+        x: 0,
+        y: 100,
+        width: 100,
+        height: 50
+      };
+    }).mockImplementation(function () {
+      return {
+        left: 0,
+        top: 0,
+        x: 0,
+        y: 200,
+        width: 100,
+        height: 50
+      };
+    });
+    let {getAllByRole, getByRole} = render(
+      <Provider theme={theme}>
+        <TagGroup maxRows={2} aria-label="tag group">
+          <Item key="1" aria-label="Tag 1">Tag 1</Item>
+          <Item key="2" aria-label="Tag 2">Tag 2</Item>
+          <Item key="3" aria-label="Tag 3">Tag 3</Item>
+          <Item key="4" aria-label="Tag 4">Tag 4</Item>
+          <Item key="5" aria-label="Tag 5">Tag 5</Item>
+          <Item key="6" aria-label="Tag 6">Tag 6</Item>
+          <Item key="7" aria-label="Tag 7">Tag 7</Item>
+        </TagGroup>
+      </Provider>
+    );
+
+    let tags = getAllByRole('gridcell');
+    expect(tags.length).toBe(1);
```

Comment:
if you log out these in the TagGroup code
```
        console.log('boundingClient tag', tag.outerHTML, width, y);
        console.log('boundingClient button', button.outerHTML, buttonWidth);
        console.log('boundingClient container', containerEnd);
        console.log('boundingClient last tag end', lastTagEnd);
```
you'll see what you need to change in the mock implementation
you're missing 'right'